src/firebase/db.js

```diff
@@ -0,0 +1,25 @@
+import {db} from "../firebase/firebase";
+
+export const addUser = async (guest) => {
+  try {
+    db.collection("guests").doc(`${guest.firstName} ${guest.lastName}`).set(guest);
+    console.log("guest added!");
+  } catch (error) {
+    console.log(error);
+  }
+};
+
+export const findUser = async (guest) => {
+  try {
+    const guests = db.collection("guests").doc(`${guest.firstName} ${guest.lastName}`);
```

Comment:
Looks like you do `${guest.firstName} ${guest.lastName}` a lot. Would it be worth making guest a class and adding a getter?